balloon/src/main/java/com/skydoves/balloon/TextViewExtension.kt

```diff
@@ -41,3 +46,41 @@ private fun fromHtml(text: String): Spanned? {
     Html.fromHtml(text)
   }
 }
+
+fun TextView.applyDrawable(vectorTextViewModel: VectorTextViewModel) {
+  val height = vectorTextViewModel.iconSize
+    ?: vectorTextViewModel.heightRes?.let { context.resources.getDimensionPixelSize(it) }
+    ?: vectorTextViewModel.squareSizeRes?.let { context.resources.getDimensionPixelSize(it) }
+
+  val width = vectorTextViewModel.iconSize
+    ?: vectorTextViewModel.widthRes?.let { context.resources.getDimensionPixelSize(it) }
+    ?: vectorTextViewModel.squareSizeRes?.let { context.resources.getDimensionPixelSize(it) }
+
+  val drawableLeft: Drawable? =
+    vectorTextViewModel.drawableLeft ?: vectorTextViewModel.drawableLeftRes
+      ?.let {
```

Comment:
```suggestion
 vectorTextViewModel.drawableLeft ?: vectorTextViewModel.drawableLeftRes?.let {
```
How about this? This is more explicit.